osx/reset.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#  This will remove the following.
+#    1. Unset environment variable: KALITE_PYTHON
+#    2. remove the .plist file, need admin
+#    3. delete the symlinked /usr/bin/kalite command, need admin
+
+KALITE_PLIST="/tmp/org.learningequality.kalite.plist"
+KALITE_PLIST_LIBRARY="/Library/LaunchAgents/org.learningequality.kalite.plist"
```

Comment:
Can you put "org.learningequality.kalite.plist" as one variable that we can re-use?  It'll be easier to maintain later.